concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -2151,32 +2151,6 @@ jobs:
                 done > config/config.sh
 
       - aggregate:
```

Comment:
Yeah, but it is handy to have it like that, don't you think so?

I won't insist to much on that, anyway, if you prefer it without it, I will revert all the merge.